assets/training/finetune_acft_hf_nlp/components/pipeline_components/nlp_multiclass/spec.yaml

```diff
@@ -37,25 +37,11 @@ inputs:
     description: model id used to load model checkpoint.
 
   # Data PreProcess parameters (See [docs](https://aka.ms/azureml/components/text_classification_datapreprocess) to learn more)
-  task_name:
-    type: string
-    optional: false
-    enum:
-      - NLPMulticlass
-    default: NLPMulticlass
-    description: Text Classification task type
-
   label_column_name:
     type: string
     optional: false
     description: label key name
 
-  sentence1_key:
-    type: string
-    optional: false
-    default: sentences
-    description: Key for `sentence1_key` in each example line. Special characters like \ and ' are invalid in the parameter value.
-
```

Comment:
what parameter is used to get column names? or do you have a fixed data format>

Reply:
We just have label_column_name. You can find the preprocessing component column name mapping in the nlp preprocess component yaml.